patches/fsevents+2.3.2.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/node_modules/fsevents/fsevents.js b/node_modules/fsevents/fsevents.js
+index f1b31c9..a75636e 100644
+--- a/node_modules/fsevents/fsevents.js
++++ b/node_modules/fsevents/fsevents.js
+@@ -50,6 +50,7 @@ function getInfo(path, flags) {
+ function getFileType(flags) {
+   if (events.ItemIsFile & flags) return "file";
+   if (events.ItemIsDir & flags) return "directory";
++  if (events.MustScanSubDirs & flags) return "directory";
```

Comment:
For more information about this change see this issue: https://github.com/vitejs/vite/issues/14043
The storybook restart issue in local was related to the dependency `fsevents` used in `chodikar` (the file watcher used in storybook). The issue has been fixed in the latest version of `fsevents` but this version seems incompatible on linux atm so we're patching the dependency with the fix in the meantime.